app/design/frontend/base/default/layout/somethingdigital/backtotop/somethingdigital_backtotop.xml

```diff
@@ -3,15 +3,13 @@
   <default>
     <reference name="head">
         <action method="addItem">
-          <type>skin_js</type>
-          <name>js/somethingdigital/back-to-top.js</name>
-        </action>
-        <action method="addItem">
           <type>skin_css</type>
           <name>css/somethingdigital/back-to-top.css</name>
         </action>
     </reference>
     <reference name="before_body_end">
+        <block type="core/template" name="back.to.top.config" template="somethingdigital/backtotop/data/config.phtml" />
```

Comment:
You could put the check here:

```
<action method="setTemplate" ifconfig="sd_backtotop/enabled">
                    <template>somethingdigital/backtotop/back-to-top.phtm</template>
                </action>
```

instead of in the template.